sdk/Trace/SpanContext.php

```diff
@@ -86,6 +86,17 @@ public static function generate(bool $sampled = false): SpanContext
         return self::fork(self::randomHex(16), $sampled);
     }
 
+    /**
+     * Creates a new sampled context with random trace
+     *
+     * @param boolean $sampled Default: false
+     * @return SpanContext
+     */
+    public static function generateSampled(): SpanContext
```

Comment:
Just decided to add this to increase readability so one doesn't have to guess what is "true" in `SpanContext::generate(true);`